src/celengine/selection.cpp

```diff
@@ -116,12 +116,12 @@ string Selection::getName(bool i18n) const
     {
     case Type_Star:
         {
-            return fmt::sprintf("#%d", star()->getCatalogNumber());
+            return fmt::sprintf("#%d", star()->getIndex());
         }
 
     case Type_DeepSky:
         {
-            return fmt::sprintf("#%d", deepsky()->getCatalogNumber());
+            return fmt::sprintf("#%d", deepsky()->getIndex());
```

Comment:
as above.

I also doubt that we need curly braces here, could you remove them?